web/blog/components/ImgCap.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+const ImgCap = (props) => {
+  return (
+    <div>
+      <p align="center">
+        <figure>
+          <img
+            // alt="Waspello - no optimistic UI update"
```

Comment:
I guess you wanted to provide some context here with these comments, explain the specific prop? This feels like a bit non-standard way to do it. If you want, I think you could use propTypes to provide more info for each prop, like type, is it required and you could also add a comment for each one. Here is an example of how it works: https://stackoverflow.com/questions/45398709/react-proptypes-in-stateless-functional-component

Reply:
Hey @matijaSos , i didn't add these comments intentionally, just in case thing's didn' go well, so i feel they can be removed safely.